.circleci/config.yml

```diff
@@ -87,7 +87,18 @@ jobs:
             export STITCH_CONFIG_DIR=/tmp/${CIRCLE_PROJECT_REPONAME}
             source /usr/local/share/virtualenvs/tap-tester/bin/activate
             pip install pymongo==4.4.0
-            run-test --tap=tap-mongodb tests
+            selected_tests=$(circleci tests glob "tests/*{test_mongodb}*")
+            echo -e "Tests running in this batch:\n$selected_tests\n"
+            echo $selected_tests > ./tests-to-run
+            if [ -s ./tests-to-run ]; then
+              for test_file in $(cat ./tests-to-run)
+              do
+                run-test --tap=tap-mongodb $test_file
+              done
+            else
+              echo "ERROR - File not found or file is empty!" >&2
+              exit 5
+            fi
```

Comment:
If this is to enable parallelism, I think it's incomplete. 

[From the latest test run](https://circleci.com/api/v1.1/project/github/singer-io/tap-mongodb/2042/output/108/0?file=true&allocation-id=64ba190a3670246c9ba1d497-0-build%2F7B12EA0), you can see that `Tests running in this batch` gets printed once with all of the tests. Also in CircleCI, [the workflow](https://app.circleci.com/pipelines/github/singer-io/tap-mongodb/1616/workflows/621cac12-66b9-40d1-a78f-5e447a26a31f/jobs/2042/parallel-runs/0) doesn't show more than one node running the tests

Reply:
@luandy64 this was not to enable parallelism as the builds were running in batch already.
This change was done to include MongoDB version 6.0 for tests & remove MongoDB version 4.2 as it has reached End of Life.
But, we can execute them in parallel & I can make this change, if it's a good idea.